src/state-manager/TransactionConsensus.ts

```diff
@@ -1811,7 +1811,20 @@ class TransactionConsenus {
       nestedCountersInstance.countEvent('consensus', 'get_tx_timestamp found tx timestamp in cacheById')
       return tsReceipt
     }
-
+    if (Math.abs(cycleCounter - CycleChain.newest.counter) > 1 ){
```

Comment:
can we please add some comments around logic to understand what's happening?

Reply:
Do you mean in the code or here?

Reply:
Allowing some tx from cycle counter +1 and -1 range.